client/components/keyed-suggestions/index.jsx

```diff
@@ -7,6 +8,34 @@ import { cosineSimilarity } from 'calypso/lib/trigram';
 
 import './style.scss';
 
+const TAXONOMY_TRANSLATIONS = {
+	feature: i18n.translate( 'Feature', {
```

Comment:
Looks like the translation doesn't work in this way.

![image](https://user-images.githubusercontent.com/13596067/201268919-8261373d-d37a-4edd-8555-a7933efa60b3.png)

We have to define it inside the component or make a function to return the taxonomy as we cannot get locale when we import this file.

Reply:
You're right, thanks for pointing this out.